guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -0,0 +1,20 @@
+Adam Růžička
+Ashish Humbe
+Chris Roberts
+Evgeni Golov
+Girija Soni
+Ian Ballou
+Jeremy Lenz
+Leos Stejskal
+Lucy Fu
+Lukas Magauer
+Maria Agaphontzev
+Markus Bucher
+Nagoor Shaik
+Partha Aji
+Quinn James
+Samir Jha
+Sayan Das
+Stejskal Leos
```

Comment:
This is Leos, also present on line 8.

Perhaps @stejskalleos should fix his git config if he shows up as both :)

Reply:
Huh, I'll take a look at where this came from ...